spread-shellcheck

```diff
@@ -137,7 +137,7 @@ def checksection(data, env: Dict[str, str]):
 def checkfile(path):
     logging.debug("checking file %s", path)
     with open(path) as inf:
-        data = yaml.load(inf)
+        data = yaml.load(inf, Loader=yaml.CSafeLoader)
```

Comment:
Any reason over just using `yaml.safe_load`?

Reply:
Started out with that, @chipaca suggested, CSafeLoader since it can be slightly faster. In my totally unscientific benchmark, there was a tiny difference, so went with that.